caluma/mutation.py

```diff
@@ -125,7 +128,7 @@ def __init_subclass_with_meta__(
         _meta.return_field_type = return_field_type
 
         input_fields = yank_fields_from_attrs(input_fields, _as=InputField)
-        super(SerializerMutation, cls).__init_subclass_with_meta__(
+        super(Mutation, cls).__init_subclass_with_meta__(
```

Comment:
Is there a reason why we're using python2-style `super()` here?

Reply:
Changed